lib/metasploit/framework/login_scanner/ldap.rb

```diff
@@ -11,19 +11,32 @@ class LDAP
         include Metasploit::Framework::LDAP::Client
         include Msf::Exploit::Remote::LDAP
 
+        # TODO: Verify if we need this additional metadata: https://github.com/rapid7/metasploit-framework/blob/master/lib/metasploit/framework/login_scanner/http.rb#L15-L23
```

Comment:
We'll need to make sure that the ldaps functionality is implemented correctly here, similar to the above link